testing/oauth_server.ts

```diff
@@ -90,19 +90,24 @@ export function launchOAuthServerFlow({
       new Error(`OAuth provider returns error ${oauthResponse.status} ${oauthResponse.text}`);
     }
 
-    const {access_token: accessToken, refresh_token: refreshToken, ...data} = await oauthResponse.json();
+    const responseBody = await oauthResponse.json();
+    const tokenContainer = nestedResponseKey ? responseBody[nestedResponseKey] : responseBody;
+    const {access_token: accessToken, refresh_token: refreshToken, ...data} = tokenContainer;
 
     return {accessToken, refreshToken, data};
   };
   const serverContainer = new OAuthServerContainer(callback, afterTokenExchange, port);
 
-  const authorizationUri = withQueryParams(authorizationUrl, {
-    scope,
```

Comment:
Nit, if you wanted to still do this inline, I think you could do `[scopeParamName || 'scope']: scope`.

Reply:
I tried this, but something very strange happens:

![Screen Shot 2022-05-18 at 2 15 43 PM](https://user-images.githubusercontent.com/84040450/169157339-725d004f-64bf-49bf-923e-28ec97943760.png)

![Screen Shot 2022-05-18 at 2 15 52 PM](https://user-images.githubusercontent.com/84040450/169157345-b7896604-47e0-42cd-926b-7e62ee5afab2.png)

So I'm reverting to the verbose version.

Reply:
Oh this is just specific to the Slack pack though because there's still `additionalParams: {user_scope: '...'}` in that manifest, right?

The only difference is the ordering--in the verbose one you're writing user_scope after additionalParams, in the terse one you're writing user_scope before additionalParams. Maybe you could just move `[scopeParamName || 'scope']: scope,` to the bottom if you want it to take precedence over `additionalParams`.

Reply:
OHHHH yes I need to remove `additionalParams` explicitly, DOH